python/paddle/incubate/optimizer/functional/__init__.py

```diff
@@ -14,5 +14,6 @@
 
 from .bfgs import minimize_bfgs  # noqa: F401
 from .lbfgs import minimize_lbfgs  # noqa: F401
+from .lbfgs_v2 import LBFGS  # noqa: F401
 
-__all__ = ['minimize_bfgs', 'minimize_lbfgs']
+__all__ = ['minimize_bfgs', 'minimize_lbfgs', 'LBFGS']
```

Comment:
class API should be `paddle.incubate.optimizer.LBFGS/BFGS`，can not be placed under `paddle.incubate.optimizer.functional`

Reply:
done